feature-libs/order/core/facade/order-history-extended-view.service.ts

```diff
@@ -0,0 +1,260 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { StateUtils, UserIdService } from '@spartacus/core';
+import {
+  Consignment,
+  ConsignmentTracking,
+  ConsignmentView,
+  Order,
+  OrderHistory,
+  OrderHistoryList,
+  OrderHistoryListView,
+  OrderHistoryView,
+  OrderView,
+  ReturnRequestList,
+} from '@spartacus/order/root';
+import { Observable, of, using, combineLatest } from 'rxjs';
+import { auditTime, filter, map, switchMap, tap } from 'rxjs/operators';
+import { OrderHistoryConnector } from '../connectors';
+import { OrderActions } from '../store';
+import {
+  getOrderById,
+  getOrderByIdState,
+} from '../store/selectors/order-by-id.selector';
+import {
+  getConsignmentTrackingById,
+  getConsignmentTrackingByIdState,
+} from '../store/selectors/order-group.selectors';
+import { OrderHistoryService } from './order-history.service';
+import { OrderReturnRequestService } from './order-return-request.service';
+
+@Injectable()
+export class OrderHistoryExtendedViewService {
+  protected orderReturnRequestService = inject(OrderReturnRequestService);
+  protected orderHistoryConnector = inject(OrderHistoryConnector);
+  protected store = inject(Store);
+  protected userIdService = inject(UserIdService);
+  protected orderHitsoryService = inject(OrderHistoryService);
+
+  clearOrderList(): void {
+    this.orderHitsoryService.clearOrderList();
+  }
+  getOrderDetailsWithTracking(
```

Comment:
[MINOR]
For clarity of purpose, this method can be made `protected` as it's not used in any other class.

Reply:
shall we leave this method public? as this is like getter method for the new ngrx entity?